spk/radarr/PLIST

```diff
@@ -0,0 +1,3 @@
+lnk:lib/libsqlite3.so
```

Comment:
This doesn't seem correct, as it's already included via `cross/sqlite/PLIST`. Please remove this, we'll do the same for the Sonarr package (or maybe you can create a PR for that as well)